galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager/src/main/java/dev/galasa/selenium/codesnippet_01.md

```diff
@@ -30,13 +32,15 @@ page.clickElementById("search_button_homepage");
 ```
 
 The code showcases different actions which can be performed on a WebPage interface to interact with different WebElements on the Browser. These WebElements are selected using a range of different techniques which allows the tester flexibility in how they are selected.
+</details>
 
-### Extracting WebPage information
+<details><summary>Extracting WebPage information</summary>
 
 ```
 WebElement element = page.findElementById("search_button_homepage");
 String pageTitle = page.getTitle();
 String pageSource = page.getPageSource();
 ```
 
-The code shows different ways of gaining information about the WebPage to be tested against. Extracting the title is a very simple way of checking if the WebDriver is on the correct page and making sure that a WebElement is found.
\ No newline at end of file
+The code shows different ways of gaining information about the WebPage to be tested against. Extracting the title is a very simple way of checking if the WebDriver is on the correct page and making sure that a WebElement is found.
```

Comment:
WebPage -> web page as we are not talking about the interface but a page on the web

Reply:
Thanks have updated